src/actions/index.js

```diff
@@ -0,0 +1,12 @@
+ export const REMOVE_BOOK = 'REMOVE_BOOK'
+ export const CREATE_BOOK = 'CREATE_BOOK'
+
+export const createBook = (book) => ({
+  type: CREATE_BOOK,
+  book
```

Comment:
We have to generate an id for the book here as well. It can be done using `Math.random()` or you can use a library like [shortid](https://www.npmjs.com/package/shortid) to generate it.